packages/components/shared/src/controllers/selection.ts

```diff
@@ -20,6 +20,9 @@ export class SelectionController<T = any> {
    */
   #selection = new Set<T>();
 
+  //** The currently highlighted row */
```

Comment:
Typo in the comment. Also could you make the comment more generic? The idea behind the `SelectionController` is that it is not grid specific.